app/Infrastructure/Exceptions/UserNotFoundException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace App\Infrastructure\Exceptions;
```

Comment:
We are using hexagonal architecture in this poyecto. Do u think this exception is on this layer??